### ISSUE TITLE
Error for simple code completion

### ISSUE THREAD

**Issue by jwbullard** [Exafunction/codeium.nvim#254]
This may not be a new issue, but I've looked around on stackexchange and reddit without success.

I just installed the plugin for neovim using lazy.nvim.  I did the authorization (seemed to go okay) and :CmpStatus now shows codeium under the list of "ready source names".  But when I try the canonical example by opening a new fib.py file and typing
```
def fib(n):
```
there is no suggested autocompletion.  Nothing happens at all.  I have attached the error log for codeium and I'm hoping someone might have a suggestion about what I'm doing wrong.

[codeium.log.log](https://github.com/user-attachments/files/17437594/codeium.log.log)

**Comment by lucabenaventew** [comment]
Yes, Im currently having the same error, It doesnt suggest any completion.
Also using nvim. Hope they can fix this soon.

**Comment by dimfeld** [comment]
For now you might consider trying my branch https://github.com/dimfeld/codeium.nvim/tree/all-fixes.

Despite the branch name, the only changes are that it incorporates my virtual text PR and that it's running the latest language server version. It's working fine for me right now, can't make any guarantees but hope it helps.

**Comment by knubbbe** [comment]
I had the same problem. But after some testing it started working when I manually added the option `enable_cmp_source = true` to `require("codeium").setup({})`.
Like this: `require("codeium").setup({ enable_cmp_source = true })`
I also have codeium manually added in nvim-cmp as a source:
```lua
sources = cmp.config.sources({
    { name = "codeium", max_item_count = 40, priority = 1500 },
})
```
Not sure if thats needed though.

**Comment by jwbullard** [comment]
Thanks!  I made those modifications as you suggested.  They didn’t seem to help but they didn’t hurt either so I am keeping them.

Jeffrey W Bullard PhD | FACerS | FWAS
Zachry Department of Civil and Environmental Engineering
Department of Materials Science and Engineering
Texas A&M University
DLEB 503F | TAMU 3136
College Station, Texas 77843-3137


On October 28, 2024 at 6:14:33 AM, Marcus Melvinson ***@***.******@***.***>) wrote:

I had the same problem. But after some testing it started working when I manually added the option enable_cmp_source = true to require("codeium"). setup({}). Like this: require("codeium"). setup({ enable_cmp_source = true }) I also have codeium
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.

ZjQcmQRYFpfptBannerEnd

I had the same problem. But after some testing it started working when I manually added the option enable_cmp_source = true to require("codeium").setup({}).
Like this: require("codeium").setup({ enable_cmp_source = true })
I also have codeium manually added in nvim-cmp as a source:

sources = cmp.config.sources({
    { name = "codeium", max_item_count = 40, priority = 1500 },
})

Not sure if thats needed though.
